test/test_Bassa_endpoint.py

```diff
@@ -0,0 +1,35 @@
+import unittest
+import requests
+
+headers = {
+'Host': 'localhost:5000',
```

Comment:
Although all these headers are returned in browser request, all of them are not needed for the backend to function. Let's keep `Accept`, `Content-Type`, `Host`, `Origin` and remove the rest

Reply:
done ..